sdk/cognitivelanguage/ai-language-text/src/transforms.ts

```diff
@@ -502,6 +525,30 @@ export function transformAnalyzeBatchResults(
           projectName,
         };
       }
+      case "ExtractiveSummarizationLROResults": {
+        const { results } = actionData as ExtractiveSummarizationLROResult;
+        const { modelVersion, statistics } = results;
+        return {
+          kind: "ExtractiveSummarization",
+          results: transformDocumentResults(docIds, results),
+          completedOn,
+          ...(actionName ? { actionName } : {}),
+          ...(statistics ? { statistics } : {}),
+          modelVersion,
+        };
+      }
+      case "AbstractiveSummarizationLROResults": {
+        const { results } = actionData as AbstractiveSummarizationLROResult;
+        const { modelVersion, statistics } = results;
+        return {
+          kind: "ExtractiveSummarization",
```

Comment:
Not sure if this is correct?

Reply:
Good catch, addressed in https://github.com/Azure/azure-sdk-for-js/pull/23208/commits/18572596257ebf264dbc975d99261716de3ba55a